mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/composeScreen/DebitCardScreen.kt

```diff
@@ -0,0 +1,204 @@
+package org.mifos.mobilewallet.mifospay.bank.fragment
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusDirection
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusProperties
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.text.TextRange
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import org.mifos.mobilewallet.mifospay.bank.presenter.DebitCardUiState
+import org.mifos.mobilewallet.mifospay.bank.presenter.DebitCardViewModel
+import org.mifos.mobilewallet.mifospay.bank.ui.SetupUpiPinActivity
+import org.mifos.mobilewallet.mifospay.common.Constants
+import org.mifos.mobilewallet.mifospay.designsystem.component.MfLoadingWheel
+import org.mifos.mobilewallet.mifospay.theme.MifosTheme
+import org.mifos.mobilewallet.mifospay.utils.Toaster.showToast
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun DebitCardScreen(
```

Comment:
Read this https://medium.com/androidbytesensei/write-testable-compose-function-a381330b7d15

Reply:
preview is generated